girder/utility/server.py

```diff
@@ -64,13 +65,11 @@ def configureServer(test=False, plugins=None, curConfig=None):
     mimetypes.add_type('application/font-woff', '.woff')
 
     curConfig.update(appconf)
+    if (mode):
```

Comment:
Parens not necessary.

Reply:
Can you tell I've been writing a lot of JS? :grin: